### PR TITLE
disallow provisioning api to be enabled for groups only

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by jnfrmarks** [comment]
@DeepDiver1975 

Is there any backwards compatibility issue we need to consider? This also has a doc implication (I thought of these after you did the PR :) )

File: apps/provisioning_api/appinfo/info.xml [hunk 1]
```diff
@@ -19,4 +19,7 @@
 	<documentation>
 		<admin>admin-provisioning-api</admin>
 	</documentation>
+	<types>
+		<filesystem/>
```
Comment (MorrisJobke) [inline apps/provisioning_api/appinfo/info.xml]: This looks like a weird workaround. Can you please add a comment that clarifies this. In the longterm we should add an option to allow an app to be enabled for specific users only.

Comment (MorrisJobke) [inline apps/provisioning_api/appinfo/info.xml]: Or an option to disallow selective enabling.

Comment (DeepDiver1975) [inline apps/provisioning_api/appinfo/info.xml]: > Or an option to disallow selective enabling.

would indeed be better - PR welcome :wink:

**Comment by DeepDiver1975** [comment]
> Is there any backwards compatibility issue we need to consider?

not really from my understanding - in case the app would have been enabled previously for certain groups only it either was not working and we would have received bug reports. 

But even this risk is low - the app was previously only available to enterprise customers.

In order to avoid problems with migrations in the future I'm all in to apply this fix for 8.0

File: apps/provisioning_api/appinfo/info.xml (hunk 1 above)
Comment (MorrisJobke) [inline apps/provisioning_api/appinfo/info.xml]: Oh look ... weird magic happened. Somehow there is now a comment ;)

**Comment by MorrisJobke** [comment]
Works :+1:

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please :gun:

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please :gun:

**Comment by DeepDiver1975** [comment]
maybe we try a rebase?

**Comment by PVince81** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Just give jenkins another chance ;)

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please :gun:

**Comment by DeepDiver1975** [comment]
breaking jenkins can be ignored -> merge - THX guys

**Pull request merged**